tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -916,15 +923,19 @@ def _for_each_trt_node(self, graph_def, fn):
 
   # TODO(laigd): provide a utility function to optimize a ConcreteFunction and
   # use it here (b/124792963).
-  def convert(self, num_calibration_runs=None, calibration_input_map_fn=None):
+  def convert(self, num_calibration_runs=None, calibration_input_fn=None):
     """Convert the input SavedModel in 2.0 format.
 
     Args:
       num_calibration_runs: number of runs of the graph during calibration.
-      calibration_input_map_fn: a function that returns inputs for the converted
-        tf_function to be calibrated.
-        Example: ```
-        def input_map_fn(): return input1, input2, input3 ```
+      calibration_input_fn: a function that returns input data as a list or
+        tuple, which will be used to execute the converted signature for
+        calibration. All the returned input data should have the same shape.
```

Comment:
Why should all the inputs have the same shapes?

Reply:
Fixed the one for build(). For calibration this is required since it can only support single engine today.